octo-task-service/src/main/java/com/revature/octo/task/repository/TaskRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.revature.octo.task.repository;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import com.revature.octo.task.model.Task;
+
+@Repository
+public interface TaskRepository extends CrudRepository<Task, Integer>{
+	
+	List<Task> findByIdIn(List<Integer> taskList);
```

Comment:
Do we need findByIdIn??
Later we will need updateTask method. Should just be in controller using repo.save(T)